src/main/java/com/example/demo/service/implementation/ProjectServiceImplementation.java

```diff
@@ -68,35 +85,115 @@ public ResponseEntity<ProjectResponseDto> updateProject(Long id, ProjectRequestD
 	public ResponseEntity<ProjectResponseDto> removeProject(Long id) {
 		// TODO Auto-generated method stub
 		Project project=projectdao.findById(id).orElseThrow(()->new ResourceNotFoundException("No any Project registered with this id ::"+id));
-	project.getEmployee().forEach(e->project.removeEmployee(e));
-	project.getDepartment().forEach(d->d.removeProject(project));
-	projectdao.deleteById(id);
+//	project.getEmployee().forEach(e->project.removeEmployee(e));
+		List<Employee>eList=project.getEmployee();
+		for(int i=0;i<eList.size();i++)
```

Comment:
It would be better to use foreach loop here, using 'i' or 'j' variables not recommended. If you want to use index based loops, try to use some meaningful names, e.g. employeeIndex or eIndex. Sometimes it better to have big name.

Reply:
Hi Suraj,

Thank you for your feedback. I understand the importance of using meaningful names for loop variables. However, due to the risk of a concurrent modification exception, I can't use a `foreach` loop in this case, so I need to stick with the normal `for` loop.

That said, I will certainly use more descriptive names for the loop variables, such as `employeeIndex` or `eIndex`, to enhance readability.

Thanks again for your valuable input.